test.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/bash -eu
 
-docker build -f Dockerfile.test -t memtar_test .
-docker run --rm -v "$(pwd):/src" memtar_test rake spec
+echo "==> Starting test.sh"
+
+echo "==> Removing Gemfile.lock"
+rm -f Gemfile.lock
+
+echo "==> Docker Run"
+docker run --rm \
```

Comment:
do we want this to default to true?

Reply:
@neil-k-zero , I'm not actually sure why it's set to default to true.